components/Text.re

```diff
@@ -153,13 +156,23 @@ module Md = {
          the markdown, otherwise we will just pass children down
          without any modification */
       let codeElement =
-        switch (lang) {
-        | "re"
-        | "reason" =>
+        switch (flavourContext.flavour) {
+        | Reason
+        | OCaml =>
+          let raw = children->Obj.magic;
```

Comment:
For string casts you can use `Unsafe.elementToString` (prevents problems when raw is not used with string based apis)

Reply:
Because later we use `ReactDOMRe.createElementVariadic` we have to stick to `Obj.magic` here :/